gli4py/glinet.py

```diff
@@ -310,30 +322,71 @@ async def wireguard_client_list(self) -> dict:
                 )
         return configs
 
-    async def wireguard_client_state(self) -> dict:
+    async def wireguard_client_state(self) -> list:
         """
-        {"rx_bytes":0,"ipv6":"","tx_bytes":0,"domain":"vpn.example.com","group_id":7707,"port":51820,"name":"TheOracle","peer_id":1341,"status":0,"proxy":True,"log":"","ipv4":""}

```

Comment:
Can you check if this is true for modern firmware - as it seems you are using tunnel_id and ned peer_id

Reply:
What exactly do you want me to verify? Just so I understand.  
But in-case you're referring to the comment being outdated, you'd be right.   
Here's the full output of calling that endpoint on 4.8.1 (with some redactions)
```json
{
	"id": 7,
	"jsonrpc": "2.0",
	"result": {
		"status_list": [
			{
				"enabled": true,
				"type": "wireguard",
				"tx_bytes": 24953488020,
				"peer_name": "PEER NAME HERE",
				"domain": [
					"XXX.XXX.XXX.XXX"
				],
				"group_id": 1234,
				"via": "wgclient1",
				"port": 51820,
				"tunnel_id": 10,
				"rx_bytes": 49725356924,
				"peer_id": 5678,
				"status": 1,
				"name": "Primary Tunnel",
				"ipv4": "XXX.XXX.XXX.XXX"
			}
		],
		"mode": 0
	}
}
```
Should I update the comment with the new response?